programmers/문자열-내림차순으로-배치하기.js

```diff
@@ -0,0 +1,5 @@
+function solution(s) {
+    let sArr = s.split(``);
+    let answer = sArr.sort().reverse().join(``)
+    return answer;
```

Comment:
변수병을 좀 더 신경쓸걸 여기서는 배열이 아닌데 Arr은 옳지 못함
변수를 여러번 선언하며 나눌필요없이 한번에 쓸수있다.
```suggestion
   return s.split('').sort().reverse('').join('');
```